pkg/kritis/review/review.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package review
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/crd/securitypolicy"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	"github.com/grafeas/kritis/pkg/kritis/util"
+	"github.com/grafeas/kritis/pkg/kritis/violation"
+	"k8s.io/api/core/v1"
+)
+
+type Reviewer struct {
+	client   metadata.MetadataFetcher
+	vs       violation.Strategy
+	validate securitypolicy.ValidateFunc
+}
+
+func New(client metadata.MetadataFetcher, vs violation.Strategy, validate securitypolicy.ValidateFunc) Reviewer {
+	return Reviewer{
+		client:   client,
+		vs:       vs,
+		validate: validate,
+	}
+}
+
+// Review reviews a given images against ImageSecurityPolicies and return error
```

Comment:
Some grammar/spelling issues here.

How about just: "Review a set of images against a set of policies"?